onnxruntime/core/graph/graph_proto_serializer.cc

```diff
@@ -46,10 +46,16 @@ void GraphViewerToProto(const GraphViewer& graph_view,
     std::unordered_set<std::string> current_scope_initializer_set;
 
     auto& initializers = graph_view.GetAllInitializedTensors();
+    std::vector<std::string> const_inits;
```

Comment:
@scottmckay do you have any comments/concerns about this change for GraphViewerToProto()?
Intel would like the initializers to always be inserted in the same order for the same ModelProto. ONNX spec doesn't require it but it doesn't seem like a harmful change.

Reply:
What's the reason the hashing function can't apply the sort? 

I can't think of a reason why it would matter. AFAIK the cost of a std::sort when saving a model shouldn't be critical. If this is required there should be comments added in the code explaining why we're doing this extra step of sorting though so it doesn't get accidentally removed.

Reply:
The hash is applied on the serialized model externally outside the scope of ORT. Hence we need to maintain the order of initializers. 

We shall add comments  stating the same. Thanks